### PR TITLE
Fstests subprocess kvm xfstests

### PR DESCRIPTION
**Pull request by alexandrasp**
- Add a method to check if kvm-xfstests is available in the client environment.
- Include kernelci library as a requirement.

### CODE REVIEW

**Comment by alexandrasp** [comment]
Now, after installing the dependency (`kernelci`), we will be able to run `run` and `check` commands from the client side using:

`python3 src/fstests/runner.py --settings=config/kernelci.conf run --db-config=staging.kernelci.org`
`python3 src/fstests/runner.py --settings=config/kernelci.conf check --db-config=staging.kernelci.org`

File: src/fstests/runner.py [hunk 1]
```diff
@@ -41,6 +50,13 @@ def run(self):
             self._db.unsubscribe(sub_id)
             return True
 
+class cmd_check(Command):
+    help = 'Check KVM-xfstests environment'
+    args = [Args.db_config]
+    opt_args = [Args.verbose]
+
+    def __call__(self, configs, args):
+        return FstestsRunner(configs, args).check_kvm_xfstest_env()
```
Comment (gctucker) [inline src/fstests/runner.py:59]: The `check_kvm_xfstests_env()` method is not returning anything, so please make it return `True` or `False` when there was an error.

File: src/fstests/runner.py [hunk 2]
```diff
@@ -21,6 +22,14 @@ def __init__(self, configs, args):
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.db.get_db(self._db_config, api_token)
 
+    def check_kvm_xfstest_env(self):
+        try:
+            result = subprocess.run(["kvm-xfstests --help"], check=True, shell=True)
```
Comment (gctucker) [inline src/fstests/runner.py:27]: With `shell=True` you should be able to run this without a list:
```suggestion
            result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
```

File: src/fstests/runner.py (hunk 1 above)
Comment (alexandrasp) [reply src/fstests/runner.py:59]: good point, I forgot to include this during the rebase.

File: src/fstests/runner.py (hunk 2 above)
Comment (alexandrasp) [reply src/fstests/runner.py:27]: good as well! I started this using call, but when I went to python 3.6 doc I saw the explicitly recommendation to use `run`, and probably I missed this detail about the list won't be more necessary.

File: src/fstests/runner.py [hunk 3]
```diff
@@ -21,6 +22,17 @@ def __init__(self, configs, args):
...
+    def check_kvm_xfstest_env(self):
+        try:
+            result = subprocess.run("kvm-xfstests --help", check=True, shell=True)
+            if result.returncode == 0:
+                print(f"KVM-xftests found.")
+                return True
+            else:
+                return False
+        except Exception as e:
+            print(f"Raised exception: {e}")
```
Comment (gctucker) [inline src/fstests/runner.py:34]: `return False` here?

**Review by gctucker — APPROVED** [review]

**Pull request merged**